optimum/commands/onnxruntime/quantize.py

```diff
@@ -0,0 +1,69 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from ...onnxruntime.configuration import AutoQuantizationConfig
+from ...onnxruntime.quantization import ORTQuantizer
+
+
+def parse_args_onnxruntime_quantize(parser):
+    required_group = parser.add_argument_group("Required arguments")
+    required_group.add_argument(
+        "--onnx_model",
+        type=Path,
+        required=True,
+        help="Path to the repository where the ONNX models to quantize are located.",
+    )
+
+    optional_group = parser.add_argument_group("Optional arguments")
+    optional_group.add_argument(
+        "-o",
+        "--output",
+        type=Path,
+        help="Path to the directory where to store generated ONNX model. (defaults to --onnx_model value).",
+    )
+    optional_group.add_argument(
+        "--per_channel",
+        action="store_true",
+        help="Compute the quantization parameters on a per-channel basis.",
+    )
+
+    level_group = parser.add_mutually_exclusive_group(required=True)
```

Comment:
Same comment as the optimization level, maybe we could add the possibility to specify a path to an `ORTConfig`.